src/plugins/intel_cpu/src/transformations/tpp/x64/op/equation.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "equation.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace tpp {
+namespace op {
+
+EquationTPP::EquationTPP(const OutputVector& arguments, std::vector<OpDescTPP> op_descs) :
+                        modifier::TensorProcessingPrimitive(), ov::op::Op(arguments),
+                        m_op_descs(std::move(op_descs)) {
+    // Initialize input/output ports as memory access ports
+    std::set<size_t> ma_iport_idx;
+    size_t i = 0;
+    while (i < get_input_size())
+        ma_iport_idx.insert(ma_iport_idx.end(), i++);
```

Comment:
It's something new for me 😄 Can we make it easier just using the loop `for`?
```suggestion
    for (size_t i = 0; i < get_input_size(); ++i)
        ma_iport_idx.insert(ma_iport_idx.end(), i);
```

Reply:
Ok, it's completely equivalent